capabilities.tf

```diff
@@ -0,0 +1,72 @@
+// This file is replaced by code-generation using 'capabilities.tf.tmpl'
```

Comment:
I assume this file is the same as what we have in our other application modules?

Reply:
Almost, a task doesn't support load balancers.